src/common/components/header/index.js

```diff
@@ -7,7 +7,7 @@ import styles from './header.css';
 
 const wordmark = 'https://assets.ubuntu.com/v1/d45097a4-snapcraft.io-logotype.svg';
 
-class Header extends Component {
+export class Header extends Component {
```

Comment:
The top container doesn't need to own _all_ the data. Containing elements can be nested. 

Consider the repository list component. A container could be used to own the data the table needs. It could be dropped inside the dashboard page container, or any other page as necessary. It would separate presentational and functional concerns.

I'm arguing for binding _one or more levels above_ presentational elements, not at the _top level_.